packages/build/bin/run-mocha.js

```diff
@@ -58,6 +61,17 @@ function run(argv, options) {
     mochaOpts.splice(lang, 2);
   }
 
+  // Set `--parallel` for `@loopback/*` packages
```

Comment:
I am not sure if this is a good idea, because it affects code outside our monorepo too. Two examples:

- Example projects created via `lb4 example <name>`. Besides the extra complexity related to parallel testing, I am concerned about the impact on performance.

- External repositories like https://github.com/strongloop/loopback4-example-shopping. I am not sure how much work is required to support parallel testing in the shopping example and I would rather not block upgrading to the latest `@loopback/build` version until we can figure out parallel testing.

I am proposing to enable parallel testing by changing `npm run mocha` script to add `--parallel` flag to mocha CLI options.

Reply:
See e.g. b44c7ee398e2aecdf89a5

```diff
-    "mocha": "node packages/build/bin/run-mocha --lang en_US.UTF-8 --timeout 5000 \"packages/*/dist/__tests__/**/*.js\" \"extensions/*/dist/__tests__/**/*.js\" \"examples/*/dist/__tests__/**/*.js\" \"packages/cli/test/**/*.js\" \"packages/build/test/*/*.js\"",
+     "mocha": "node packages/build/bin/run-mocha --lang en_US.UTF-8 --timeout 15000 --parallel \"packages/*/dist/__tests__/**/*.js\" \"extensions/*/dist/__tests__/**/*.js\" \"examples/*/dist/__tests__/**/*.js\" \"packages/cli/test/**/*.js\" \"packages/build/test/*/*.js\"",
```

Having said that, I think it's time to move flags from `run-mocha` CLI args into the monorepo-level `.mocharc.js` file. That way they are always applied, for example when running a subset of tests manually via `node packages/build/bin/run-mocha path/to/some/files`.